agent/main.go

```diff
@@ -83,6 +87,7 @@ func setupflags() {
 	klog.ClearLogger()
 
 	flag.StringVar(&namespace, "namespace", "default", "Namespace in the management cluster where you would like to register this host")
+	flag.Int64Var(&certExpiryDuration, "certExpiryDuration", registration.ExpirationSeconds, "Duration for the expiration of the host certificates")
```

Comment:
Can we also specify the unit of duration in flag description?

What is the expiry value you expect customers would use? Kubernetes certificates generally have an expiry of an year. If we expect the same range, do you think users would prefer to specify expiry duration in days instead of seconds.

Reply:
Updated the description.